diff --git a/node_modules/pagedjs-cli/dist/browser.js b/node_modules/pagedjs-cli/dist/browser.js
index df4f9b3..4831186 100644
--- a/node_modules/pagedjs-cli/dist/browser.js
+++ b/node_modules/pagedjs-cli/dist/browser.js
@@ -30161,7 +30161,7 @@
 
 			if (typeof node.prelude === "undefined" ||
 					node.prelude.type !== "AtrulePrelude" ) {
-				return;
+				return media;
 			}
 
 			csstree.walk(node.prelude, {
@@ -30773,7 +30773,9 @@
 		onRule(ruleNode, ruleItem, rulelist) {
 			let selector = csstree.generate(ruleNode.prelude);
 			if (selector.match(/:(first|last|nth)-of-type/)) {
-				
+				if (selector.match(/:(before|after)/)) {
+					return;
+				}
 				let declarations = csstree.generate(ruleNode.block);
 				declarations = declarations.replace(/[{}]/g,"");
 
@@ -30828,6 +30830,9 @@
 		onRule(ruleNode, ruleItem, rulelist) {
 			let selector = csstree.generate(ruleNode.prelude);
 			if (selector.match(/\+/)) {
+				if (selector.match(/:(before|after)/)) {
+					return;
+				}
 
 				let declarations = csstree.generate(ruleNode.block);
 				declarations = declarations.replace(/[{}]/g,"");
